doc/tutorial.rst

```diff
@@ -229,5 +229,5 @@ Step 3: Build your project
 ==========================
 
 Now you can just build your project by running
-``dpkg-buildpackage -us -uc -b``.
+``( deactivate ; dpkg-buildpackage -us -uc -b )``.
```

Comment:
I'm slightly cautious here: we don't run any virtualenvs prior in this documentation. Now it is entirely possible, that people have an virtualenv activated, which would screw up the build, but not sure if we should default to deactivation? Not entirely sure on my take on this. Maybe separate note to tell people to deactivate if they are already in dh-virtualenv?

Reply:
this is a secure command variant always succeeding for all people. what's called a winner… ;)

and the effects of not having it when you need it are mostly silent, nasty, and not easily debugged.

Reply:
That's true!